### ISSUE TITLE
sign up procedure has some problems

### ISSUE THREAD

**Issue by nicb** [SMERM/EMUForm#47]
We need to make a `capybara` test upon failing sign up procedures. There are no `flash` notices, thus it is difficult to understand what is going on. Furthermore: it's a bit funny. Signing in requires an e-mail but then the e-mail is not reported in the inner form.

**Comment by micsuperfly** [comment]
send me reset èassword..has ssome problems

**Comment by nicb** [comment]
> send me reset èassword..has ssome problems

thank you @micsuperfly I'll open a new issue for this (this is a separate issue).

**Comment by nicb** [comment]
Something about it can be found [here](http://stackoverflow.com/questions/4101641/rails-devise-handling-devise-error-messages)

**Comment by grammaton** [comment]
@nicb but all the `notice` we have into controllers works fine.
maybe simply we need to write the `notice` for log in.

**Comment by grammaton** [comment]
@nicb after merging master on gs-cap and the pull of your fixes the `LOG IN` button not works

**Comment by nicb** [comment]
> @nicb after merging master on gs-cap and the pull of your fixes the LOG IN button not works

Yes I am checking. Unfortunately most of the `javascript`s we have included in our `application` layout page are broken and/or behave very erratically. So here is what I am currently trying to do:

1. fix the javascript errors that appear on browser consoles the best I can
1. remove all the unnecessary javascripts from the `sign up`/`sign in` pages.

**Comment by grammaton** [comment]
this is what we have into `emufest.org/_site/assets`
```
.
├── css
│   └── main.css
├── emufest_assets
├── fonts
│   ├── FontAwesome.otf
│   ├── fontawesome-webfont.eot
│   ├── fontawesome-webfont.svg
│   ├── fontawesome-webfont.ttf
│   └── fontawesome-webfont.woff
└── js
    ├── plugins
    │   ├── jquery.fitvids.js
    │   ├── jquery.magnific-popup.js
    │   ├── respond.js
    │   ├── responsive-nav.js
    │   └── search.js
    ├── scripts.min.js
    └── vendor
        ├── html5shiv.min.js
        ├── jquery-1.9.1.min.js
        ├── modernizr-2.7.1.custom.min.js
        └── respond.min.js

5 directories, 17 files
```
